### PR TITLE
[docs-only] Language review on new IDM section

### PR DESCRIPTION
**Pull request by EParzefall**
Just some minor wording/spelling fixes.

### CODE REVIEW

**Review by mmattel — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]

File: docs/extensions/idm/_index.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ of the LDAP tree is `o=libregraph-idm`. IDM gives LDAP write permissions to a si
 [boltdb](https://github.com/etcd-io/bbolt) file `idm/ocis.boltdb` inside the oCIS base data directory.
 
 Note: IDM is limited in its functionality. It only supports a subset of the LDAP operations (namely BIND, SEARCH, ADD, MODIFY, DELETE).
-Also IDM currently does not do any Schema Verification (e.g. structural vs. auxiliary Objectclasses, require and option Attributes,
-Syntax Checks, ...). So it's not meant as a general purpose LDAP server.
+Also IDM currently does not do any schema verification (e.g. structural vs. auxiliary object classes, require and option attributes,
```
Comment (micbar) [inline docs/extensions/idm/_index.md:25]: hmm, in LDAP protocol `objectclass` is written without a space.

**Review by micbar — COMMENTED** [review]
Small comment

File: docs/extensions/idm/_index.md (hunk 1 above)
Comment (EParzefall) [reply docs/extensions/idm/_index.md:25]: > hmm, in LDAP protocol `objectclass` is written without a space.

Yes, but generally speaking like in this case, it should be object class. Example from: 
https://ldapwiki.com/wiki/ObjectClass
ObjectClass [attribute](https://ldapwiki.com/wiki/Attribute) specifies the object classes of an entry, which (among other things) are used in conjunction with the controlling schema to determine the permitted attributes of an entry.

Comment (micbar) [reply docs/extensions/idm/_index.md:25]: fine for me.

**Pull request merged**